bin/cylc-get-host-metrics

```diff
@@ -116,22 +98,23 @@ def main(parser, options):
     sys.exit()
 
 
-def run_command_string(command_string):
+def run_command(cmd):
     """Return the result from running a command string in a local shell.
 
     Return stdout if command successful, else raise CalledProcessError.
     """
-    command_sequence = command_string.split()
     # Note: can replace function w/ 'subprocess.check_output()' when drop v2.6.
-    process = Popen(command_sequence, stdin=open(os.devnull), stdout=PIPE,
```

Comment:
After the removal of the calls to functions of `os` like this one here, the import `os` can be removed :point_up: